### PR TITLE
Fix typo

### LINKED ISSUE #4: Typo in "Homepage" key name
The homepage data from the flatpak remote is supplied in a dictionary key named "hompage". This should be "homepage".

### PR DESCRIPTION
**Pull request by isantop**
Changes the name of the "homepage" key in remotes from "hompage" to "homepage". This will make it easier to develop for in the future.

Fixes #4

### CODE REVIEW

**Comment by jackpot51** [comment]
Why are there 3 version releases in commits on this?

**Comment by isantop** [comment]
A tiny glitch in the automatic versioning. If it's a problem, I can delete this branch and re-create it

**Comment by jackpot51** [comment]
You just need to squash it

**Comment by isantop** [comment]
Was planning on squashing prior to merging anyway.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,22 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [0.4.1](https://github.com/pop-os/pyflatpak/compare/v0.0.2...v0.4.1) (2020-02-04)
+
+### 0.0.2 (2020-02-04)
```
Comment (jackpot51) [inline CHANGELOG.md:7]: Does this need to be here?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,22 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [0.4.1](https://github.com/pop-os/pyflatpak/compare/v0.0.2...v0.4.1) (2020-02-04)
```
Comment (jackpot51) [inline CHANGELOG.md:5]: Should this be comparing 0.4.0?

**Pull request closed without merging**